apps/avifenc.c

```diff
@@ -188,6 +188,8 @@ static avifInputFile * avifInputGetNextFile(avifInput * input)
     }
     return &input->files[input->fileIndex];
 }
+
+// Returns whether the next avifInputGetNextFile() call will succeed.
```

Comment:
Renaming this function something like "avifInputHasMoreFiles" would also be helpful by making the name more self-explanatory.

Reply:
"HasMoreFiles" is slightly incorrect, as this function will return true while you are still reading out of the middle of a multiframe y4m.

Reply:
I see why you chose the name you chose. Perhaps if anything, `avifInputGetNextFile()` needs a new name.

Reply:
Thanks for taking a look! Note that this pull request does not rename the function. It only adds a comment.